backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -0,0 +1,272 @@
+dislayed is simply "Hjørring".  The members are, apart from the obvious
+ones, to be interpreted like this:
+
+* ``locations`` are addresses and can be of type ``DAR``, ``EMAIL``
+  and ``PHONE``. The list contains a list of addresses that are to be
+  interpreted as couples, *(type, value)*.
+* ``employees`` correspond to ``engagement`` in MO. They are four-tuples,
+  *(name, UUID, engagement type, job function)*.
+* ``departments`` are the children of the current node, i.e. the
+  subsections etc. They are couples, *(name, UUID)*.
+* ``associated`` correspond to ``association`` in MO. They are
+  four-tuples, *(name, UUID, association type, job function)*.
+* ``managers`` correspond to ``manager`` in MO and are triplets,
+  *(manager type, name, UUID)*.
+
+
+**NOTE:** The UUIs for employees, associated and managers are the person
+UUIDs, found under the MO API's ``/e/`` section.
+
+Employees
+.........
+
+The "wrapping" with response headers etc. is like for departments. A
+typical "doc" for a person could be: ::
+
+    {
+        "uuid":["cee8800a-983d-41fa-998c-b4557d68ec35"],
+        "name":["Anna Bjerre Reidl"],
+        "locations":["PHONE",
+          "21557342",
+          "DAR",
+          "Strandgårdsvej 16, Skallerup Klit, 9800 Hjørring",
+          "EMAIL",
+          "annar@hjorring.dk"],
+        "departments":["IT-Support",
+          "d3a9e589-5be0-4d28-95af-5d24ac42a2e9",
+          "Ansat",
+          "Specialist"],
+        "managing":["Direktør",
+          "IT-Support",
+          "d3a9e589-5be0-4d28-95af-5d24ac42a2e9"],
+        "id":"e6207b7c-6204-44e2-8839-277694589883",
+        "_version_":1616933478548373504
+    }
+
+
+This record has the following non-trivial members:
+
+* ``locations`` - this works as for departments.
+* ``departments`` - corresponds to ``engagement`` in MO. These are
+  to be interpreted as four-tuples consisting of *(department name,
+  UUID, engagement type, job function)*. Departments in which the person
+  is *employed*.
+* ``associated`` - corresponds to ``association`` in MO and organized as
+  ``departments``. Departments to which the person is associated.
+  Association type might e.g. be "Konsulent".
+  (tilknyttet).
+* ``managing`` - the departments which the user is managing. These are
+  triplets consisting of *(department name, UUID, manager type)*.
+  Manager type might e.g. be "Afdelingsleder" or "Direktør".
+
+
+Subdepartments/sections
+-----------------------
+
+The names and UUIDs of the departments/sections immediately under the
+root may be found in the root doc's ``department`` member, as described
+above. 
+    
+You can access the full data (including phone numbers, etc.) for each of
+these subdepartments by doing a query for the corresponding UUID, as we
+did with the root department.
+
+
+Display results (general query)
+===============================
+
+There are two separate URLs for searching departments, basically ::
```

Comment:
```suggestion
Two separate URLs allow searching for departments or employees::
```
“Basically” seems redundent — unless you have more. Also, “there are” is passive tense…